test/per_file_coverage.sh

```diff
@@ -20,7 +20,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/signal:87.2" # Death tests don't report LCOV
 "source/common/thread:0.0" # Death tests don't report LCOV
 "source/common/watchdog:58.6" # Death tests don't report LCOV
-"source/exe:91.5"
+"source/exe:90.3"
```

Comment:
Out of curiosity, how does `--use-dyanmic-base-id` result in a coverage drop?

Reply:
Not sure myself.

Reply:
These lines become uncovered: https://github.com/envoyproxy/envoy/blob/b483d296738516839ec09ea3056f9c90be362080/source/exe/stripped_main_base.cc#L144-L145

Reply:
Ah, nice catch. @kyessenov can we perhaps write a test which does cover those lines?

Reply:
I don't understand why UDS sockets are shared in shell test. I thought bazel runs them in a container? @ravenblackx do you have some de-container setting or it happened on CI?

Reply:
I believe the sockets collide (absent dynamic-base-id) if you run the tests in the standard envoy dev container with e.g. `--runs_per_test=4`, but it was CI that failed, I wouldn't care if it was local-only.

You could make them not collide by setting `socketPath` to the test dir, except that the path to test tmpdir is typically very long and unix socket max path length is less long, so that doesn't work. You could also potentially make a tmpdir in `/tmp` for the purpose, set `socketPath` to *that*, and solve both problems, but then you're left with a potential cleanup issue if the test crashes.